docs/documentation/server_admin/topics/identity-broker/social/openshift.adoc

```diff
@@ -38,36 +38,35 @@ grantMethod: prompt <4>
 ==== OpenShift 4
 
 .Prerequisites
-. Installation of https://stedolan.github.io/jq/[jq].
-. `X509_CA_BUNDLE` configured in the container and set to `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`.
+. A certificate of the OpenShift 4 instance stored in the Keycloak Truststore.
+. A Keycloak server configured in order to use the truststore. For more information, see the https://www.keycloak.org/server/keycloak-truststore[Configuring a Truststore] {section}.
 
 .Procedure
-. Run the following command on the command line and note the OpenShift 4 API URL output.
+. Click *Clients* in the {project_name} menu.
+. Click *Identity Providers* in the {project_name} menu.
+. Enter your *Client ID*, *Client Secret*, *Valid Redirect URIs* and *Web Origins*.
```

Comment:
I'm not sure I follow. :) Why to click Clients and create a Client? There's no Client at KC side. KC needs to have IdP, Client is created in OpenShift.